### ISSUE TITLE
Autofight/v key throw launcher instead of firing a projectile

### ISSUE THREAD

**Issue by janosseboek** [crawl/crawl#2589]
**PLATFORM and OS or BROWSER:**
local tiles

**DCSS VERSION AND GAME INFORMATION:**
Crawl version 0.29-a0-723-gce39da1c22
Save file version 34.247
Compiled with GCC 7.5.0
Build platform: x86_64-suse-linux
Platform: x86_64-suse-linux
CFLAGS: -O2 -pipe -DUSE_TILE -DUSE_TILE_LOCAL -DUSE_SDL -DUSE_GL -DUSE_FT -D_REENTRANT -Wall -Wformat-security -Wundef -Wextra -Wno-missing-field-initializers -Wno-implicit-fallthrough -Wno-type-limits -Wno-uninitialized -Wno-array-bounds -Wmissing-declarations -Wredundant-decls -Wno-parentheses -Wwrite-strings -Wshadow -pedantic -Wuninitialized -Iutil -I. -isystem /usr/include/freetype2 -isystem /usr/include/SDL2 -isystem contrib/install/x86_64-suse-linux/include -DWIZARD -DASSERTS -DPROPORTIONAL_FONT="/usr/share/fonts/truetype/DejaVuSans.ttf" -DMONOSPACED_FONT="/usr/share/fonts/truetype/DejaVuSansMono.ttf" -D_REENTRANT -DCLUA_BINDINGS
LDFLAGS: -rdynamic -O2 

**DESCRIPTION:**
When firing an arrow/stone/bolt, the launcher is thrown at the enemy instead. Hilarious but game-breaking. Happens both in auto-fight and when using "v".

To recreate, make any hunter and press tab at the first enemy.

**EXTRA FILES:**
init.txt
```
remember_name = false
default_manual_training = true

crawl_dir = /home/janos/.crawl
save_dir = /home/janos/.crawl/saves/
morgue_dir = /home/janos/.crawl/morgues/
easy_confirm = none
show_travel_trail = false
autofight_stop = 75
autofight_throw = true
autofight_caught = true
```
The autofight_throw option made no difference.

**Comment by Sobieck** [comment]
This also happened to me.

**Comment by OcalaDave** [comment]
Also happening on webtiles, version 0.29-a0-723-gce39da1c22. Tried swapping in default .rc file but bug still occurs.

**Comment by riverfiend** [comment]
Same bug, webtiles 0.29-a0-723-gce39da1. Pressing tab, f, v, or F all result in "you shoot a +0 shortbow. The +0 shortbow hits the bat but does no damage!"

**Comment by PleasingFungus** [comment]
Fixed in a706738f4e6e7aac651a091c1e810357f175fb9d , thanks for reporting & sorry for the trouble!
